docs/content/development/extensions-core/s3.md

```diff
@@ -65,6 +65,23 @@ As an example, to set the region to 'us-east-1' through system properties:
 `s3:GetObject` and `s3:PutObject` are basically required for pushing/loading segments to/from S3.
 If `druid.storage.disableAcl` is set to `false`, then `s3:GetBucketAcl` and `s3:PutObjectAcl` are additionally required to set ACL for objects.
 
+### S3 authentication methods
+
+To connect to your S3 bucket (whether deep storage bucket or source bucket), Druid use the following credentials providers chain
+
+|order|type|details|
+|--------|-----------|-------|
+|1|Druid config file|Based on your runtime.properties if it contains values `druid.s3.accessKey` and `druid.s3.secretKey` |
+|2|Custom properties file| Based on custom properties file where you can supply `sessionToken`, `accessKey` and `secretKey` values|
```

Comment:
would be nice if you document 'druid.s3.fileSessionCredentials' as well or else this point is harder to understand.

Reply:
you are perfectly right, I forgot it.
i added the property and some precisions about how to set those properties

Reply:
I'm wondering is that could not be better to remove informations in /content/configuration/index.md about s3 deep storage and just let the link to s3.md page.
That would avoid information duplication, no? 
If OK, I'll just report `druid.storage.useS3aSchema` in s3.md and make the link

Reply:
sure, for the auth stuff , there are other deep storage specific properties which should still be left there.

Reply:
done. I also reported properties that were on one side but not on other (when relevant).